test/e2e/fips.go

```diff
@@ -19,13 +19,14 @@ const (
 	mFips = "99-master-fips"
 )
 
-var _ = Describe("Validate FIPS Mode", func() {
+var _ = Describe("FIPS Mode", func() {
```

Comment:
This test is duplicated based on what we just merged.  I think we should keep the FIPs machine configs within the FIPs test, and remove them out of the other test that was merged.  

Whatever you think though.

Reply:
#2449

Reply:
@bennerv agree. After merging #2449 I noticed this test also lists MachineConfigPools instead of MachineConfigs. I was going to simplify this as a follow up after merging this one.

So I suggest we do the following:

* Merge this PR with changes proposed
* I'll create a PR which:
    * Refactors `MachineConfigPools` to `MachineConfigs` in this test
    * Removes FIPS from #2449

Does it make sense?